dlr/dlr-engine/src/main/java/ai/djl/dlr/jni/LibUtils.java

```diff
@@ -24,11 +44,302 @@
 @SuppressWarnings("MissingJavadocMethod")
 public final class LibUtils {
 
+    private static final Logger logger = LoggerFactory.getLogger(LibUtils.class);
+
+    private static final String LIB_NAME = "djl_dlr";
+    private static final String NATIVE_LIB_NAME = "dlr";
+
+    private static final Pattern VERSION_PATTERN =
+            Pattern.compile("(\\d+\\.\\d+\\.\\d+(-\\w)?)(-SNAPSHOT)?(-\\d+)?");
+
     private LibUtils() {}
 
     public static void loadLibrary() {
-        // TODO implement
-        System.load("/Users/leecheng/workspace/djl/dlr/dlr-native/libdlr.dylib");
-        System.load("/Users/leecheng/.djl.ai/dlr/1.5.0-cpu-osx-x86_64/libdjl_dlr.dylib");
+        String nativePath = findNativeOverrideLibrary();
+        String jniPath;
```

Comment:
To make is simple, we should always copy jni to our cachDir

Reply:
done